### PR TITLE
Update CE blueprints to new baseline

### PR DESCRIPTION
**Pull request by PhilPJL**
Update all blueprints since they all have new contentPack/Copyright.
Moved CeContentPackContext.Load() after CacheTADefinitions.

            DiagnosticsContext.CacheTADefinitions();

            // Needs to be after CacheTADefinitions
            CeContentPackContext.Load();

### CODE REVIEW

File: Diagnostics/CommunityExpansionBlueprints/CampaignDefinition/TelemaDemoUnleashed.json [hunk 1]
```diff
@@ -1763,8 +1763,8 @@
     "sortOrder": 0,
     "unusedInSolastaCOTM": false
   },
-  "contentCopyright": "OpenGameContent",
+  "contentCopyright": "UserContent",
```
Comment (DubhHerder) [inline Diagnostics/CommunityExpansionBlueprints/CampaignDefinition/TelemaDemoUnleashed.json:1766]: I'm not sure how this should be categorized. The demo campaign is obviously not content added by CE but CE is required to make it available to be played. Will the demo have the CE icon attached to it?

**Review by DubhHerder — COMMENTED** [review]
Obviously the changes are automated and ok for CE generated content but i'm wondering what should be done with things like the telema demo campaign where a feature created by TA is just unlocked by CE rather than created be CE. 
I also dont know if there is any other content that falls into the same catagory as the telema demo in CE at the moment. It's hard to say just looking through the JSONs, we'd probably be faster asking people if they have unhidden TA features.

Not related to this PR but on the same topic , I know that i unlocked unique monsters for DM mode in DM Pro mod and as Zappa plans to hid that mod ASAP, i will be merging it with CE. I might just not unlock those TA monsters in the CE version.

File: Diagnostics/CommunityExpansionBlueprints/CampaignDefinition/TelemaDemoUnleashed.json (hunk 1 above)
Comment (PhilPJL) [reply Diagnostics/CommunityExpansionBlueprints/CampaignDefinition/TelemaDemoUnleashed.json:1766]: Ok that one needs to be fixed.  contentCopyright is set in AddToDB().  There's an existing 

            public static CampaignDefinition TelemaDemo { get; } = GetDefinition<CampaignDefinition>("TelemaDemo", "d007f98ad89d171448159b55da41f6a7");

in DatabaseHelpers, so I don't know why we're adding another @ThyWoof .

**Comment by PhilPJL** [comment]
> Obviously the changes are automated and ok for CE generated content but i'm wondering what should be done with things like the telema demo campaign where a feature created by TA is just unlocked by CE rather than created be CE. I also dont know if there is any other content that falls into the same catagory as the telema demo in CE at the moment. It's hard to say just looking through the JSONs, we'd probably be faster asking people if they have unhidden TA features.
> 
> Not related to this PR but on the same topic , I know that i unlocked unique monsters for DM mode in DM Pro mod and as Zappa plans to hid that mod ASAP, i will be merging it with CE. I might just not unlock those TA monsters in the CE version.

As I noted above no blueprints will have the contentCopyright changed unless we create them and use AddToDB().  Anything 'unlocked' shouldn't be affected.

**Review by ThyWoof — DISMISSED** [review]

File: Diagnostics/CommunityExpansionBlueprints/CampaignDefinition/TelemaDemoUnleashed.json (hunk 1 above)
Comment (DubhHerder) [reply Diagnostics/CommunityExpansionBlueprints/CampaignDefinition/TelemaDemoUnleashed.json:1766]: yeah. that's what i listed above. 

It still doesnt resolve the issue of assigning TA content with a CE icon

Comment (ThyWoof) [reply Diagnostics/CommunityExpansionBlueprints/CampaignDefinition/TelemaDemoUnleashed.json:1766]: I’m ok removing this one…

**Comment by PhilPJL** [comment]
I added an overload to AddToDB to address this issue.

**Pull request merged**